common/src/main/java/com/alibaba/fescar/common/executor/Initialize.java

```diff
@@ -0,0 +1,13 @@
+package com.alibaba.fescar.common.executor;
+
+/**
+ * @author zhangsen
+ */
+public interface Initialize {
```

Comment:
Could you please rename this method?

Reply:
rename to what ?

Reply:
@zhangthen Turn the first letter of the method name to lowercase.

Reply:
The  first letter of the  method ‘init’  which is the  only method of class  Initialize  always been   lowercase.